pom.xml

```diff
@@ -96,6 +96,13 @@
       <version>1</version>
       <optional>true</optional>
     </dependency>
+
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-engine</artifactId>
```

Comment:
`junit-jupiter-api` will be good enough here - we don't need engine in test classpath